### DISCUSSION TITLE
Downloaded Video is Black and No audio

### DISCUSSION THREAD

**Discussion by Its-Silvermoon** [aclap-dev/video-downloadhelper#1305] [Help and Q&A]
### Your question

I tried to download DRM protected MPD Video from Rigi (https://app.rigi.club/) Powerd creators course, the video was downloaded but there is no audio and a black screen is masked on the video. Is there any possibility to overcome this.
![10](https://github.com/user-attachments/assets/edb32f4a-2bca-476c-b688-b5526228e044)

### Addon & CoApp version and details

Version 9.1.0.44 (stable), target google, locale en-US. Companion App installed (v2.0.19)

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

You should have searched this forum using `drm` as your search key.  You would have learned that VDH has never, does not now, nor will ever successfully download any content protected by DRM.  To do so would be illegal.  DRM exists for the express purpose of preventing downloading of copyrighted material.  You are observing 100% expected behavior.

**Reply by GrampaWildWilly** [reply]
By the way, when you post your environment information, you should include 2 additional items, like this: Windows 11 Pro 64-bit version 23H2 build 22631.4249, Firefox 130.0.1 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20.
